diff --git a/src/Controller/TripController.php b/src/Controller/TripController.php
index ae2d292..3189a3c 100644
--- a/src/Controller/TripController.php
+++ b/src/Controller/TripController.php
@@ -52,7 +52,6 @@ final class TripController extends AbstractController
                 ->to(new Address($customer->getEmail()))
                 ->subject('Booking Confirmation for '.$trip->getName())
                 ->htmlTemplate('email/booking_confirmation.html.twig')
-                ->textTemplate('email/booking_confirmation.txt.twig')
                 ->context([
                     'customer' => $customer,
                     'trip' => $trip,
diff --git a/templates/email/booking_confirmation.txt.twig b/templates/email/booking_confirmation.txt.twig
deleted file mode 100644
index 119e611..0000000
--- a/templates/email/booking_confirmation.txt.twig
+++ /dev/null
@@ -1,10 +0,0 @@
-Hey {{ customer.name|split(' ')|first }},
-
-Get ready for your trip to {{ trip.name }}!
-
-Departure: {{ booking.date|date('Y-m-d') }}
-
-Manage your booking: {{ url('booking_show', {uid: booking.uid}) }}
-
-Regards,
-The Universal Travel Team
