diff --git a/src/Command/SendBookingRemindersCommand.php b/src/Command/SendBookingRemindersCommand.php
index 9405037..1189192 100644
--- a/src/Command/SendBookingRemindersCommand.php
+++ b/src/Command/SendBookingRemindersCommand.php
@@ -17,7 +17,7 @@ use Symfony\Component\Scheduler\Attribute\AsCronTask;
     name: 'app:send-booking-reminders',
     description: 'Send booking reminder emails',
 )]
-#[AsCronTask('# # * * *')]
+#[AsCronTask('#midnight')]
 class SendBookingRemindersCommand extends Command
 {
     public function __construct(
