diff --git a/src/Scheduler/MainSchedule.php b/src/Scheduler/MainSchedule.php
index 807efa9..653519f 100644
--- a/src/Scheduler/MainSchedule.php
+++ b/src/Scheduler/MainSchedule.php
@@ -2,6 +2,7 @@
 
 namespace App\Scheduler;
 
+use Symfony\Component\Console\Messenger\RunCommandMessage;
 use Symfony\Component\Scheduler\Attribute\AsSchedule;
 use Symfony\Component\Scheduler\RecurringMessage;
 use Symfony\Component\Scheduler\Schedule;
@@ -20,10 +21,13 @@ final class MainSchedule implements ScheduleProviderInterface
     {
         return (new Schedule())
             ->add(
-                // @TODO - Create a Message to schedule
-                // RecurringMessage::every('1 hour', new App\Message\Message()),
+                RecurringMessage::cron(
+                    '0 0 * * *',
+                    new RunCommandMessage('app:send-booking-reminders')
+                )
             )
             ->stateful($this->cache)
+            ->processOnlyLastMissedRun(true)
         ;
     }
 }
