diff --git a/src/Command/SendBookingRemindersCommand.php b/src/Command/SendBookingRemindersCommand.php
index b3879c6..9405037 100644
--- a/src/Command/SendBookingRemindersCommand.php
+++ b/src/Command/SendBookingRemindersCommand.php
@@ -17,7 +17,7 @@ use Symfony\Component\Scheduler\Attribute\AsCronTask;
     name: 'app:send-booking-reminders',
     description: 'Send booking reminder emails',
 )]
-#[AsCronTask('0 0 * * *')]
+#[AsCronTask('# # * * *')]
 class SendBookingRemindersCommand extends Command
 {
     public function __construct(
