diff --git a/tests/Functional/BookingTest.php b/tests/Functional/BookingTest.php
index 2d5bbdd..338f9df 100644
--- a/tests/Functional/BookingTest.php
+++ b/tests/Functional/BookingTest.php
@@ -61,6 +61,7 @@ class BookingTest extends KernelTestCase
                     ->assertContains('Visit Mars')
                     ->assertContains('/booking/'.BookingFactory::first()->getUid())
                     ->assertHasFile('Terms of Service.pdf')
+                    ->dd()
                 ;
             })
         ;
