diff --git a/src/Controller/TripController.php b/src/Controller/TripController.php
index 05a37e1..b94eaee 100644
--- a/src/Controller/TripController.php
+++ b/src/Controller/TripController.php
@@ -14,6 +14,7 @@ use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\DependencyInjection\Attribute\Autowire;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Mailer\Header\TagHeader;
 use Symfony\Component\Mailer\MailerInterface;
 use Symfony\Component\Mime\Address;
 use Symfony\Component\Routing\Attribute\Route;
@@ -63,6 +64,8 @@ final class TripController extends AbstractController
                 ])
             ;
 
+            $email->getHeaders()->add(new TagHeader('booking'));
+
             $mailer->send($email);
 
             return $this->redirectToRoute('booking_show', ['uid' => $booking->getUid()]);
