diff --git a/src/Webhook/EmailEventConsumer.php b/src/Webhook/EmailEventConsumer.php
new file mode 100644
index 0000000..a959c7e
--- /dev/null
+++ b/src/Webhook/EmailEventConsumer.php
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Webhook;
+
+use Symfony\Component\RemoteEvent\Attribute\AsRemoteEventConsumer;
+use Symfony\Component\RemoteEvent\Consumer\ConsumerInterface;
+use Symfony\Component\RemoteEvent\Event\Mailer\MailerDeliveryEvent;
+use Symfony\Component\RemoteEvent\Event\Mailer\MailerEngagementEvent;
+use Symfony\Component\RemoteEvent\RemoteEvent;
+
+#[AsRemoteEventConsumer('mailtrap')]
+class EmailEventConsumer implements ConsumerInterface
+{
+    /**
+     * @param MailerDeliveryEvent|MailerEngagementEvent $event
+     */
+    public function consume(RemoteEvent $event): void
+    {
+        dump($event);
+    }
+}
